RecoTracker/PixelLowPtUtilities/interface/SlidingPeakFinder.h

```diff
@@ -0,0 +1,102 @@
+#ifndef SLIDING_PEAK_FINDER_H
```

Comment:
this does not follow rule 4.1 [1]

[1] https://cms-sw.github.io/cms_coding_rules.html